tests/query_errors/new.sh

```diff
@@ -17,9 +17,10 @@ fi
 cp -r ${BASEDIR}/_template ${TARGET}
 git add ${TARGET}
 
-sed -e "s;//INSERT;//INSERT\n    ${NAME},;" ${BASEDIR}/../query_error.rs > ${BASEDIR}/tmp && mv ${BASEDIR}/tmp ${BASEDIR}/../query_error.rs
+sed -e "/^   \/\/INSERT/a\
```

Comment:
could we copy those over to all the new.sh's (or extract them into a function :D